test/e2e_node/util.go

```diff
@@ -282,7 +282,7 @@ func getCRIClient() (internalapi.RuntimeService, internalapi.ImageManagerService
 	// connection timeout for CRI service connection
 	const connectionTimeout = 2 * time.Minute
 	runtimeEndpoint := framework.TestContext.ContainerRuntimeEndpoint
-	r, err := remote.NewRemoteRuntimeService(runtimeEndpoint, connectionTimeout)
+	r, err := remote.NewRemoteRuntimeService(runtimeEndpoint, connectionTimeout, nil)
```

Comment:
Needs NewNoopTracerProvider instead of nil.

Reply:
updated